onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -1857,30 +1857,30 @@ namespace OperatorHelper
         DowncastDimensions(gsl::span(shapeData), /*out*/ m_blockShape);
 
         const uint32_t dimCount = gsl::narrow_cast<uint32_t>(m_blockShape.size());
-        m_dilations = {dimCount, 1};
-        m_pads = {dimCount * 2, 0};
-        m_strides = {dimCount, 1};
+        m_dilations.resize(dimCount);
+        std::fill(m_dilations.begin(), m_dilations.end(), 1);
+        m_pads.resize(dimCount, 0);
+        std::fill(m_pads.begin(), m_pads.end(), 0);
+        m_strides.resize(dimCount, 1);
+        std::fill(m_strides.begin(), m_strides.end(), 1);
```

Comment:
Why repeat a fill when the resize just filled it?
```suggestion
        m_dilations.resize(dimCount, 1);
        m_pads.resize(dimCount, 0);
        m_strides.resize(dimCount, 1);
```